tests/util.py

```diff
@@ -162,7 +172,16 @@ def load_tables(bq, dataset, tables):
                         field=field.name
                     )
                     break  # stop because there can only be one time partitioning field
-        with open(table.source_path, "rb") as file_obj:
+        if isinstance(table.source_path, str):
+            with open(table.source_path, "rb") as file_obj:
+                job = bq.load_table_from_file(
+                    file_obj, destination, job_config=job_config
+                )
+        else:
+            file_obj = BytesIO()
+            for row in load(*table.source_path):
+                file_obj.write(json.dumps(row).encode() + b"\n")
+            file_obj.seek(0)
```

Comment:
added support for rewriting `main_summary_v4.yaml` as ndjson, so that we could specify inputs as yaml

Reply:
Should we add something to testing docs to make devs aware of this feature?

Reply:
yes, will file follow-up PR